packages/teleport/src/console/useOnAltKeyDown.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2020 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import ConsoleContext from './consoleContext';
+
+// DIGITS defines event.key values on keyboard from 1 - 9.
+export const DIGITS = ['1', '2', '3', '4', '5', '6', '7', '8', '9'];
```

Comment:
I'm slightly leaning towards making this a function instead:

```ts
const isNumeric(event: KeyboardEvent): bool =>
  ['1', '2', '3', '4', '5', '6', '7', '8', '9'].includes(event.key);
```

Reply:
I didn't use this approach, b/c I need the index for getting the document in the docs array.

Reply:
i used `includes` on checking if its numeric though